db/changes_test.go

```diff
@@ -119,3 +120,183 @@ func _testChangesAfterChannelAdded(t *testing.T, db *Database) {
 	assertNoError(t, err, "Couldn't GetChanges")
 	printChanges(changes)
 }
+
+func _testDocDeletionFromChannelCoalescedRemoved(t *testing.T, db *Database) {
```

Comment:
What's up with the naming on this test?  Is this indicating that it should only be run against a backing Couchbase DB?  If so, I think prefixing with Couchbase might be better.